### DISCUSSION TITLE
Will Renovate container images be deleted from Docker Hub?

### DISCUSSION THREAD

**Discussion by bmaupin** [renovatebot/renovate#20987] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

We currently use the Renovate container images from Docker Hub: https://hub.docker.com/u/renovate

Given the news that [Docker is deleting Open Source organisations](https://blog.alexellis.io/docker-is-deleting-open-source-images/), will that impact the Renovate images?

**Comment by rarkins** [comment]
It is our intention at this point to pay for a Teams account so that renovate images aren't deleted. We'll also take steps to distribute our images more widely (such as GHCR and maybe ECR) and reduce risk for users.

**Reply by viceice** [reply]
image is now available at `ghcr.io/renovatebot/renovate` 🥳

**Reply by YohanSciubukgian** [reply]
They made an update on the communication, are you concerned by the change ?
https://www.docker.com/blog/we-apologize-we-did-a-terrible-job-announcing-the-end-of-docker-free-teams/

**Reply by rarkins** [reply]
I'm not sure which change you're referring to but I definitely am concerned about what's happening

**Reply by HonkingGoose** [reply]
Docker is no longer sunsetting the Free Team Plan. [^blog] We should still use multiple container registries so we can switch the defaults if one does something silly.

[^blog]: https://www.docker.com/blog/no-longer-sunsetting-the-free-team-plan/

**Comment by MindTooth** [comment]
I want to give a plug to [Quay](https://quay.io) as well.  Just noticed [vaultwarden](https://github.com/dani-garcia/vaultwarden/releases/tag/1.28.0) started hosting there.
